astro/src/content/docs/apis/_tenant-request-body.mdx

```diff
@@ -433,6 +433,14 @@ import TransactionTypes from 'src/content/docs/apis/_transaction-types.mdx';
     * `SlidingWindowWithMaximumLifetime` - the expiration is calculated from the last time the token was used, or until the <InlineField>maximumTimeToLiveInMinutes</InlineField> is reached. &nbsp; <AvailableSince since="1.46.0"/>
   </APIField>
 
+  <APIField name="tenant.jwtConfiguration.refreshTokenOneTimeUseConfiguration.gracePeriodInSeconds" type="Integer" optional defaults="0" since="1.55.0">
```

Comment:
```suggestion
  <APIField name="tenant.jwtConfiguration.refreshTokenOneTimeUseConfiguration.gracePeriodInSeconds" type="Integer" optional defaults="0" since="1.55.1">
```